ebuild-writing/eapi/text.xml

```diff
@@ -369,19 +369,29 @@ src_compile() {
 		</p>
 		<p>Example:</p>
 		<codesample lang="ebuild">
-EAPI=1
+EAPI=2
 ...
-src_install() {
-	emake DESTDIR="${D}" install || die "make install failed"
-	dodoc ChangeLog README
+src_test() {
+	if ! emake check ; then
+		eerror "Tests failed. Looking for files for you to add to your bug report..."
+		find "${S}" -type f -name '*.epicfail' -or -name '*.log' | while read a ; do
```

Comment:
Shouldn't there be parentheses around the two `-name` tests? Otherwise, non-regular `*.log` will match too.
Also, any good reason for using `-or` (which is a GNU extension) instead of POSIX `-o`?

Reply:
It was just a cheap copy-paste, I'll look into improving it.